Homework_2/Task_1.py

```diff
@@ -0,0 +1,11 @@
+# დაწერეთ პროგრამა რომელიც მიიღებს true ან false. თუ პროგრამამ მიიღო true, ეკრანზე დაბეჭდეთ “whoala”.
+
+# Let the user enter True or False
+isTrue = input("Enter True or False: ")
```

Comment:
პითონში სტანდარტია snake_case
```suggestion
is_true = input("Enter True or False: ")
```